src/analysis/normed_space/continuous_linear_map.lean

```diff
@@ -178,44 +187,38 @@ linear_equiv.to_continuous_linear_equiv_of_bounds f a a⁻¹
 
 end seminormed
 
-/-! ## The span of a single vector -/
+/-! ## The span of a single vector
 
-section seminormed
-
-variables [seminormed_add_comm_group E] [normed_space 𝕜 E]
+Note that some of the results in this section are not about normed spaces at all! -/
```

Comment:
I think this comment (and maybe the name of the PR) is a bit misleading, because really the only generalization going on here is the base ring, because if `𝕜` is a field then I think that `[module 𝕜 E] [has_bounded_smul 𝕜 E]` is mathematically the same as `[normed_space 𝕜 E]`.
I'm not saying this is a bad change at all, any zero-cost generalization is good to have, but I feel like this comment is a bit confusing because it seems to imply a much larger generality.

Reply:
Otherwise LGTM. Thanks!
bors d+

Reply:
This comment is a remnant of an earlier version of this PR which really did do as advertised! Removed.